website/docs/seo.md

```diff
@@ -14,20 +14,24 @@ Docusaurus supports search engine optimization in a variety of ways.
 
 ## Global metadata {#global-metadata}
 
-Provide global meta attributes for the entire site through the [site configuration](./configuration.md#site-metadata). The metadata will all be rendered in the HTML `<head>` using the key-value pairs as the prop name and value.
+Provide global meta and / or link tags for the entire site through the [site configuration](./configuration.md#site-metadata). The metadata and / or link tags will all be rendered in the HTML `<head>` using the key-value pairs as the prop name and value.
 
 ```js title="docusaurus.config.js"
 module.exports = {
   themeConfig: {
     metadata: [{name: 'keywords', content: 'cooking, blog'}],
     // This would become <meta name="keywords" content="cooking, blog"> in the generated HTML
+    linkTags: [
+      {rel: 'monetization', content: 'https://ilp.uphold.com/LwQQhXdpwxeJ'},
```

Comment:
`<link rel="monetization" >` tags encode the payment endpoint inside  the href attribute

Reply:
Good catch! Fixed